src/Aquifer.Jobs/appsettings.json

```diff
@@ -28,19 +28,20 @@
-            "ZHS": "Then translate to Simplified Chinese given the following conditions: When translating make sure to use full shape for all punctuation (e.g. '、' for ',', and '）' for ')'. Always replace quotes with proper usage of 「」. When translating any text that looks like a reference to a Bible verse or Bible passage do not put a space between the book and the number. When translating a list of html span elements that have Bible verses or passages inside them, always replace delimiting characters (e.g. ';', ',', '.') such that any list of spans with Bible chapters is delimited by '，', and any list of spans with Bible verses is delimited by '、'. For example, 'cp. 1 Kgs 1:31, 33, 2:1, 2 Ch 33:14' should translate to '参王上1:31、33，2:1；代下33:14', 'see Job 18:6, 18; 19:1' should translate to '见伯18:6、18，19:1', and 'Pss 19:1-6; 97:11; 104:19-20; Isa 60:19-20' should translate to '詩19:1-6，97:11，104:19-20；賽60:19-20'.",
-            "ZHT": "Then translate to Traditional Chinese given the following conditions: When translating make sure to use full shape for all punctuation (e.g. '、' for ',', and '）' for ')'. Always replace quotes with proper usage of 「」. When translating any text that looks like a reference to a Bible verse or Bible passage do not put a space between the book and the number. When translating a list of html span elements that have Bible verses or passages inside them, always replace delimiting characters (e.g. ';', ',', '.') such that any list of spans with Bible chapters is delimited by '，', and any list of spans with Bible verses is delimited by '、'. For example, 'cp. 1 Kgs 1:31, 33, 2:1, 2 Ch 33:14' should translate to '參王上1:31、33，2:1；代下33:14', 'see Job 18:6, 18; 19:1' should translate to '見伯18:6、18，19:1', and 'Pss 19:1-6; 97:11; 104:19-20; Isa 60:19-20' should translate to '詩19:1-6，97:11，104:19-20；賽60:19-20'."
+        "HtmlBasePrompt": "You receive HTML and then return the HTML with an altered version of the text therein. Never change the formatting of the HTML elements or attributes. For example, if text has an <em> tag around it or a <span> tag with attributes, that tag and its attributes should stay with the text even if it is simplified or moved within the text. Keep the HTML tags and attributes no matter what. Never add a missing html element (i.e. if something starts with <p> but the end is missing </p>, don't add it). Never remove '&nbsp;'.",
+        "LanguageSpecificTextImprovementPromptAppendixByLanguageIso6393CodeMap": {
+            "ENG": "Simplify the text for a global audience. Aim for an 8th grade reading level. Add parenthetical explanations if needed. Keep sentences under 25 words. Use the active voice whenever natural. Spell out abbreviations, except BC and AD.",
```

Comment:
Can we add "Avoid using contractions." somewhere in here please.

Reply:
[Done](https://github.com/BiblioNexusStudio/aquifer-server/pull/589/commits/5f4a2e193c67884892c4feac73b71f4288445b39).